web/src/routes/(user)/albums/[albumId]/+page.svelte

```diff
@@ -557,14 +532,12 @@
               <section class="pt-24">
                 <AlbumTitle id={album.id} albumName={album.albumName} {isOwned} />
 
-                <!-- ALBUM SUMMARY -->
-                {#if album.assetCount > 0}
-                  <span class="my-2 flex gap-2 text-sm font-medium text-gray-500" data-testid="album-details">
-                    <p class="">{getDateRange()}</p>
-                    <p>·</p>
-                    <p>{album.assetCount} items</p>
-                  </span>
-                {/if}
+                <AlbumSummary
+                  {assetCount}
+                  {album}
+                  oldestAsset={$assetStore.assets.at(-1)}
+                  newestAsset={$assetStore.assets.at(0)}
```

Comment:
This is the list of assets from the loaded time buckets for the album, not every asset in the album. Unless I am mistaken this will only show the date range for loaded assets.